src/docs/contributing.md

```diff
@@ -0,0 +1,111 @@
+# Contributing
+
+In the first place, thank you for your interest in contributing! 🙏
+
+## Development
```

Comment:
I used soft link to mirror the file content.
I have tried other approaches, but they did not work as expected.

Reply:
There are links between `CONTRIBUTING.md` and `RELEASING.md`. These either work when looking at the file on GitHub or in generated docs. Not both.

The same thing is true with other doc files, but the issue is not so grave since the files in repository root are expected to be consumed outside of the doc website. 

It would be nice to have all links working, but I can live without it for files in `src`. It seems sort of broken for files ion the repo root though.

Reply:
OK, I changed the links, so in the root MD files, all links have absolute URLs and prefer GitHub over the rendered docs.